test_eintr Python test failing intermittently

`test_eintr` has been failing intermittently on our FVTR test suite, like on #1866. On most cases, re-running the build was enough to get the test to pass, so this is likely another timeout-related issue. This needs further investigation.

Output from a failed run:
```
test test_eintr failed -- Traceback (most recent call last):^M
  File "/home/tcbot/bot-slave/at_opt/at-next-15.0-0-alpha/lib64/python3.9/test/test_eintr.py", line 33, in test_all^M
    script_helper.assert_python_ok("-u", tester, "-v")^M
  File "/home/tcbot/bot-slave/at_opt/at-next-15.0-0-alpha/lib64/python3.9/test/support/script_helper.py", line 156, in assert_python_ok^M
    return _assert_python(True, *args, **env_vars)^M
  File "/home/tcbot/bot-slave/at_opt/at-next-15.0-0-alpha/lib64/python3.9/test/support/script_helper.py", line 142, in _assert_python^M
    res.fail(cmd_line)^M
  File "/home/tcbot/bot-slave/at_opt/at-next-15.0-0-alpha/lib64/python3.9/test/support/script_helper.py", line 70, in fail^M
    raise AssertionError("Process return code is %d\n"^M
AssertionError: Process return code is 1^M
command line: ['/home/tcbot/bot-slave/at_opt/at-next-15.0-0-alpha/bin/python3', '-X', 'faulthandler', '-I', '-u', '/home/tcbot/bot-slave/at_opt/at-next-15.0-0-alpha/lib64/python3.9/test/eintrdata/eintr_tester.py', '-v']^
```

--- a/fvtr/python/python.exp
+++ b/fvtr/python/python.exp
@@ -139,7 +139,9 @@ if { "$pyver" == "2.6" } {
 		-j$CORES -x $common_exclude_tests \
 		test_asyncgen \
 		test_compileall \
+		test_eintr \
 		test_faulthandler \
+		test_ftplib \
 		test_gdb \
 		test_httpservers \
 		test_imaplib \
@@ -151,6 +153,8 @@ if { "$pyver" == "2.6" } {
 		test_site \
 		test_smtplib \
 		test_socket \
+		test_ssl \
+		test_subprocess \
 		test_threading \
 		test_time"
 } else {

